### PR TITLE
Changing flake8 to have max-line-length of 115

### PR DESCRIPTION
**Pull request by ollie-iterators**
## Type of Changes

|     | Type                   |
| --- | ---------------------- |
|    | :bug: Bug fix          |
|    | :sparkles: New feature |
| ✓   | :hammer: Refactoring   |
|    | :scroll: Docs          |

## Description

Changing flake8 to have a max-line-length of 115 because there are issues with changing it to 103. This makes the code easier to understand.

Refs #XXXX

Closes #XXXX

### CODE REVIEW

**Comment by ollie-iterators** [comment]
Is there anything that needs to be done for this PR to be merged?

File: pylint/testutils/_primer/primer_compare_command.py [hunk 1]
```diff
@@ -128,7 +128,8 @@ def _create_comment_for_package(
             assert not self.packages[package].url.endswith(
                 ".git"
             ), "You don't need the .git at the end of the github url."
-            comment += f"{self.packages[package].url}/blob/{new_messages['commit']}/{filepath}#L{message['line']}\n"
+            comment += f"{self.packages[package].url}"
+            comment += f"/blob/{new_messages['commit']}/{filepath}#L{message['line']}\n"
```
Comment (Pierre-Sassoulas) [inline pylint/testutils/_primer/primer_compare_command.py:132]:
```suggestion
            comment += (
                f"{self.packages[package].url}"
                f"/blob/{new_messages['commit']}/{filepath}#L{message['line']}\n"
            )
```

**Review by Pierre-Sassoulas — DISMISSED** [review]
Thank you for the cleanup ! It's hard to know when the MR is ready if you push new commits periodically and don't tell us when it's ready ;)

**Comment by Pierre-Sassoulas** [comment]
The pre-commit fail seems genuine and not caused by my suggestion.

**Comment by ollie-iterators** [comment]
Strange, when I check out the file that has an error, 'pylint/config/config_initialization.py', it says that it only has 124 lines, and the pre-commit error is about lines 127 and 128.

**Comment by Pierre-Sassoulas** [comment]
This seem like a pre-commit.ci bug, maybe due to bad caching. It works locally too.

**Review by Pierre-Sassoulas — APPROVED** [review]

**Pull request merged**